code/modules/mob/living/carbon/human/human.dm

```diff
@@ -287,7 +287,7 @@
 
 
 /mob/living/carbon/human/Topic(href, href_list)
-	if(usr.canUseTopic(src, BE_CLOSE, NO_DEXTERY))
+	if(usr.canUseTopic(src, BE_CLOSE, !NO_DEXTERY))

```

Comment:
Doing this will allow monkeys and simple animals and stuff to remove glass shards from people.